### PR TITLE
Mv 436 mobile questionnaire

### PR DESCRIPTION
**Pull request by pkrucz00**
Mobile layout for the questionnaire. Enjoy.

### CODE REVIEW

File: assets/src/features/leaving-page/components/Questionnaire.tsx [hunk 1]
```diff
@@ -37,7 +38,10 @@ const Questionnaire: FC<QuestionnaireProps> = ({ onSubmitClick }) => {
   const {isSmartphone} = useSmartphoneViewport();
 
   return (
-    <form aria-label="questionnaire" className="flex flex-col items-center justify-center gap-y-10 sm:gap-y-20">
+    <form aria-label="questionnaire" className={
+      clsx("flex flex-col items-center justify-center",
+      "gap-y-10 sm:gap-y-20",
+      isSmartphone && "mb-36")}>
```
Comment (pkrucz00) [inline assets/src/features/leaving-page/components/Questionnaire.tsx]: https://i.imgflip.com/52xeoo.png

TBH - I thought there is some more elegant solution than putting a fixed element on the button of the page and adding margin to the sibling element. 

I would consider `sticky` element to be a nice solution, but it comes with it's own problems (the element has width of the parent element, not the viewport, doesn't stick to the button etc.).

For simplicity - I used this simple yet sufficient solution.

File: assets/src/features/leaving-page/icons/🚀.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        d="M126.064 43.0331C115.502 44.6312 78.0055 53.75 77.9899 102.197L102.327 116.155C144.135 91.6787 133.657 55.0945 129.867 45.2142C129.591 44.4656 129.059 43.8372 128.364 43.4386C127.669 43.04 126.855 42.8965 126.064 43.0331Z"
+        fill="#D2CFFD"
+      />
+      <path
+        d="M108.602 77.4435C111.122 78.889 114.345 78.0312 115.8 75.5275C117.255 73.0239 116.391 69.8225 113.871 68.377C111.351 66.9316 108.128 67.7894 106.673 70.293C105.218 72.7967 106.081 75.998 108.602 77.4435Z"
+        fill="#001A72"
+      />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
```
Comment (ostatni5) [inline assets/src/features/leaving-page/icons/🚀.tsx:24]: Fix properties in svg to jsx compatible
```suggestion
        fillRule="evenodd"
        clipRule="evenodd"
```

File: assets/src/features/shared/icons/Logo.tsx
Comment (ostatni5) [inline assets/src/features/shared/icons/Logo.tsx:1]: Hmm, now we have 3 files with logos. And naming can be confusing.
image

**Review by ostatni5 — APPROVED** [review]
The code looks ok.

**Pull request merged**